src/test/java/com/emc/object/s3/S3JerseyUrlConnectionV4Test.java

```diff
@@ -2,10 +2,17 @@
 
 import com.emc.object.ObjectConfig;
 import com.emc.object.s3.jersey.S3JerseyClient;
+import com.emc.object.s3.request.PutObjectRequest;
+import com.emc.util.RandomInputStream;
 import com.sun.jersey.client.urlconnection.URLConnectionClientHandler;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.InputStream;
 import java.net.URI;
 
-public class S3JerseyUrlConnectionV4Test extends S3JerseyUrlConnectionTest {
+public class S3JerseyUrlConnectionV4Test extends S3JerseyClientV4Test {
```

Comment:
S3JerseyUrlConnectionV4Test should be V4 version of S3JerseyUrlConnectionTest, why changing it to inherit from S3JerseyClientV4Test ?